src/style/app-fixed.module.css

```diff
@@ -4020,30 +4020,160 @@
 
 /* -- GeneralSettings.tsx -- */
 
+.cardSettings {
+  border-radius: 0.5rem;
+  margin-bottom: 1.5rem;
+  border: none;
+  box-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.075);
+  height: 800px;
+  width: 588px;
+}
+
+.cardHeaderStyle {
+  border-top-left-radius: 0.25rem;
+  border-top-right-radius: 0.25rem;
+  padding: 1.25rem 1rem 1rem 1rem;
+  border-bottom: 3px solid var(--cardHeader-border);
+  background-color: #eaebef;
+  padding: 16px;
+}
+
+.userupdatediv {
+  padding: 25px 16px;
+  background: var(--white-color);
+}
+
+.maskGroupButton {
+  width: 20px;
+  height: 20px;
+  color: #1778f2;
+}
+
+.customTextarea {
+  min-height: 100px;
+  background-color: #f1f3f6;
+  color: #737373;
+}
+
+.textFields {
+  background-color: #f1f3f6;
+  color: #737373;
+  margin-bottom: 1rem;
+  width: auto;
+}
+
+.descriptionTextField {
+  background-color: #f1f3f6;
+  color: #737373;
+  margin-bottom: 1rem;
+  width: auto;
+  resize: none;
+  height: 100px;
+}
+
 /* -- DeleteOrg.tsx -- */
 
+.DeleteOrgCard {
+  border-radius: 0.5rem;
+  margin-left: 1rem;
+  box-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.075);
+  border: none;
+  height: 280px !important;
+  width: 404px !important;
+}
 .settingsBody {
   margin: 2.5rem 0;
 }
 
+.deleteCardHeader {
+  background-color: #eaebef;
+  border-top-left-radius: 1.5rem;
+  border-top-right-radius: 1.5rem;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+}
+
+.btnDelete {
+  background-color: #eaebef;
+  border: #000;
+}
+
+.btnDelete:hover {
+  background-color: #eaebef;
+  color: #000 !important;
+}
+.btnDelete:active {
+  background-color: #eaebef !important;
+  color: #000 !important;
+}
+
 .deleteButton {
-  background-color: var(--delete-button-bg);
-  color: var(--delete-button-color);
-  border: none;
-  padding: 5px 20px;
-  display: flex;
-  align-items: center;
-  margin-top: 20px;
-  margin-right: auto;
-  margin-left: auto;
-  gap: 8px;
+  background-color: #f8d6dc;
+  color: #c8102e;
+  border: 1px solid #c8102e;
+  padding: 8px 16px;
+  cursor: pointer;
+  display: block;
+  margin: 10px auto;
+  border-radius: 4px;
+  width: 365px;
+  height: 42px;
+  text-align: center;
 }
 
-.deleteButton:is(:hover, :active) {
-  background-color: var(--delete-button-color) !important;
+.deleteButton:hover,
+.deleteButton:active,
+.deleteButton:focus {
+  background-color: #f8d6dc !important;
+  color: #c8102e;
+  box-shadow: 2.5px 2.5px 2.5px var(--dropdown-shadow);
+  outline: none;
+}
+
+.btnConfirmDelete {
+  background-color: #f8d6dc;
+  color: #c8102e;
```

Comment:
Use a variable name that matches the function of this element. There are many examples at the top of this file